### ISSUE TITLE
keyBoardNav no longuer supported 

### ISSUE THREAD

**Issue by djey64** [react-bootstrap-table/react-bootstrap-table2#433]
Hi,
keyBoardNav was supported in react-bootstrapt-table, but take no effect in table2.
It would be great to have this awesome feature back in table2
Thank's a lot

**Comment by AllenFang** [comment]
@djey64 yes, we are going to release `1.0.0` with a lots of legacy feature, like bootstrap4, row expand, table search and export csv. In the next, I will start to implement the keyboard navigation, thanks
